data/builtInFeatures.js

```diff
@@ -0,0 +1,125 @@
+// https://github.com/zloirock/core-js
+
+const es2015 = {
+  // "es6.typed/array-buffer": "typed arrays / ",
+  "es6.typed.data-view": "typed arrays / DataView",
+  "es6.typed.int8-array": "typed arrays / Int8Array",
+  "es6.typed.uint8-array": "typed arrays / Uint8Array",
+  "es6.typed.uint8-clamped-array": "typed arrays / Uint8ClampedArray",
+  "es6.typed.int16-array": "typed arrays / Int16Array",
+  "es6.typed.uint16-array": "typed arrays / Uint16Array",
+  "es6.typed.int32-array": "typed arrays / Int32Array",
+  "es6.typed.uint32-array": "typed arrays / Uint32Array",
+  "es6.typed.float32-array": "typed arrays / Float32Array",
+  "es6.typed.float64-array": "typed arrays / Float64Array",
+
+  "es6.map": "Map",
+  "es6.set": "Set",
+  "es6.weak-map": "WeakMap",
+  "es6.weak-set": "WeakSet",
+  // proxy not implemented
+
+  "es6.reflect.apply": "Reflect / Reflect.apply",
+  "es6.reflect.construct": "Reflect / Reflect.construct",
+  "es6.reflect.define-property": "Reflect / Reflect.defineProperty",
+  "es6.reflect.delete-property": "Reflect / Reflect.deleteProperty",
+  "es6.reflect.get": "Reflect / Reflect.get",
+  "es6.reflect.get-own-property-descriptor": "Reflect / Reflect.getOwnPropertyDescriptor",
+  "es6.reflect.get-prototype-of": "Reflect / Reflect.getPrototypeOf",
+  "es6.reflect.has": "Reflect / Reflect.has",
+  "es6.reflect.is-extensible": "Reflect / Reflect.isExtensible",
+  "es6.reflect.own-keys": "Reflect / Reflect.ownKeys",
+  "es6.reflect.prevent-extensions": "Reflect / Reflect.preventExtensions",
+  "es6.reflect.set": "Reflect / Reflect.set",
+  "es6.reflect.set-prototype-of": "Reflect / Reflect.setPrototypeOf",
+
+  "es6.promise": "Promise",
+  "es6.symbol": "Symbol",
+
+  "es6.symbol": "well-known symbols / Symbol.hasInstance",
+  "es6.symbol": "well-known symbols / Symbol.isConcatSpreadable",
+  "es6.symbol": "well-known symbols / Symbol.iterator",
+  "es6.symbol": "well-known symbols / Symbol.match",
+  "es6.symbol": "well-known symbols / Symbol.replace",
+  "es6.symbol": "well-known symbols / Symbol.search",
+  "es6.symbol": "well-known symbols / Symbol.species",
+  "es6.symbol": "well-known symbols / Symbol.split",
+  "es6.symbol": "well-known symbols / Symbol.toPrimitive",
+  "es6.symbol": "well-known symbols / Symbol.toStringTag",
+  "es6.symbol": "well-known symbols / Symbol.unscopables",
```

Comment:
Or maybe to prevent hardcoding just use it with RegExp:
For `s6.symbol`: `/(^|\s)Symbol(\.\w+)?$/` will match `Symbol` and `/ Symbol.method`.
Or `/Array.prototype.(keys|entries)$/` for `es6.array.iterator`.
But It might cause conflicts in the future. Needs really accurate RegExps.